app/controllers/projects_controller.rb

```diff
@@ -1,5 +1,14 @@
 class ProjectsController < ApplicationController
   def index
     @presenter = ProjectsPresenter.new(params)
+    @props = {
+      tags: tags
+    }
+  end
+
+  private
+
+  def tags
+    Project.all.pluck(:tags).flatten.compact.uniq.sort
```

Comment:
As we discussed, it's possible there's a better way to model this - I'm envisioning a `Team` model rather than a tag string column. But yeah totally optional and for a future PR.

Reply:
Not going to change models just yet, but agreed, `tags` is misleading for how we use it.